### PR TITLE
If uses layer keywords

### PR DESCRIPTION
**Pull request by ismailsunni**

### CODE REVIEW

**Comment by ismailsunni** [comment]
@lucernae can you review it, since you are working on the code that I modified. Other reviewers are welcome.

File: safe/impact_functions/bases/classified_rh_classified_ve.py [hunk 1]
```diff
@@ -44,10 +44,8 @@ def __init__(self):
     # pylint: disable=W0221
     def hazard(self, value):
         ImpactFunction.hazard.fset(self, value)
-        self.set_up_hazard_layer(value)
```
Comment (timlinux) [inline safe/impact_functions/bases/classified_rh_classified_ve.py]: Can you refactor set_up to setup

File: safe/storage/safe_layer.py [hunk 2]
```diff
@@ -137,3 +138,17 @@ def is_qgsvectorlayer(self):
         :rtype: bool
         """
         return isinstance(self.layer, QgsVectorLayer)
+
+    def qgis_vector_layer(self):
+        """Get QgsVectorLayer if the layer param is a vector storage layer (
```
Comment (timlinux) [inline safe/storage/safe_layer.py]: First line docstring should be 80 chars terminated by full stop.

**Comment by timlinux** [comment]
LGTM

**Pull request merged**